### ISSUE TITLE
ActionCable: Unsupported data type 'ActionDispatch::Request::Session'

### ISSUE THREAD

**Issue by lhoBas** [appsignal/appsignal-ruby#1313]
Hi team 👋 We're seeing

```
[ERROR] appsignal: Sample data 'session_data': Unsupported data type 'ActionDispatch::Request::Session' received: #<ActionDispatch::Request::Session:0x626b0 not yet loaded>
```
errors logged in our ActionCable processes, which I believe is the result of `.to_h` [missing on the `.session` call here](https://github.com/appsignal/appsignal-ruby/blob/8fb8b93af873735a33d9c9440260fd9afe9dd12b/lib/appsignal/hooks/action_cable.rb#L55) ([as is done for the Rack implementation](https://github.com/appsignal/appsignal-ruby/blob/8fb8b93af873735a33d9c9440260fd9afe9dd12b/lib/appsignal/rack.rb#L96)).

**Comment by tombruijn** [comment]
Hi @lhoBas, that does seem broken. I'm working on a fix.

**Comment by lhoBas** [comment]
Thanks for the swift fix 🙌

**Comment by tombruijn** [comment]
It's been released in Ruby gem 4.1.2 👍
